src/components/Calender.vue

```diff
@@ -1,38 +1,49 @@
 <template>
-	<section v-if="listToDisplay != undefined && listToDisplay.size > 0" id="calender">
-		<component 
+	<section
+		v-if="listToDisplay != undefined && listToDisplay.size > 0"
+		id="calender"
+	>
+		<component
 			v-for="calVal in calender"
-			:is="calVal.isPadding ? 'div' : 'a'" 
+			:is="calVal.isPadding ? 'div' : 'a'"
 			:key="calVal.date"
 			:class="{
-				'today': isToday(calVal.date),
-				'padding': calVal.isPadding,
-				'date': true,
+				today: isToday(calVal.date),
+				padding: calVal.isPadding,
+				date: true,
 			}"
 			:title="(filmExist(calVal.date)) ? listToDisplay.get(calVal.date)!.name : ''"
 			:href="(filmExist(calVal.date)) ? `https://letterboxd.com${listToDisplay.get(calVal.date)!.slug}` : ''"
 			target="_blank"
 		>
 			<template v-if="!calVal.isPadding">
 				<h3>
-					{{ calVal.date + 1 }}
+					<div class="number">{{ calVal.date + 1 }}</div>
+					<div class="title">
+						{{ (filmExist(calVal.date)) ? listToDisplay.get(calVal.date)!.name : '' }}
+					</div>
```

Comment:
The use of `div`s here feels wrong. I would sooner use a `span` with `display: block` applied, but I think at that point we have to ask why are these both wrapped up in the same `h3` tag? The title of the movie, while a title, isn't (imo) the title of this HTML block. The date number is the title, and the film title is a piece of data within the block. Starting with a separate `p` (or even `dl`!) tag would make more sense and make a lot of the hacky CSS not necessary. It would also separate the two by tag, so you won't need to rely on the class names as much, if at all.

You've also probably noticed the font for the title is defaulting to Times New Roman. This is because the date numbers use a subsetted font that only contains digits. I would recommend switching to the body font set elsewhere.

Finally, the reason I didn't bother adding the film titles was because of the spacing issues you're running into. If you to work through it, be my guest. However, another option would be to set up [VueTippy](https://vue-tippy.netlify.app/), and have the titles appear in a nicely designed tooltip on hover. We use this for [Film Frequency](https://github.com/GoodbyteCo/Letterboxd-Film-Frequency).

Reply:
Ah ya the tooltip seems smart actually. What I really want is someway to have the font size responsively just take up the remaining width of the date square next to the date number and then disappear at a certain size but all the mechanisms for this involve like jquery or something very overkill